packages/charts/react-charts-preview/library/src/components/Legends/Legends.types.ts

```diff
@@ -132,9 +132,9 @@ export interface LegendsProps {
   legends: Legend[];
 
   /**
-   * Additional CSS class(es) to apply to the legneds component.
+   * Additional CSS class(es) to apply to the legends component.
    */
-  className?: string;
+  className?: LegendsStyles;
```

Comment:
revert this change?

Reply:
We need to keep the type as LegendStyles so that the classNames can be applied to all the sub-components as well. Otherwise, it can be applied to root only.